executor/index_merge_reader.go

```diff
@@ -320,6 +316,22 @@ func (w *partialTableWorker) fetchHandles(ctx context.Context, fetchCh chan<- *l
 	}
 }
 
+func (w *partialTableWorker) handleHandlesFetcherPanic(ctx context.Context, resultCh chan<- *lookupTableTask) func(r interface{}) {
```

Comment:
Actually, we do not need to define `handleHandlesFetcherPanic` as the member functions for  `partialTableWorker` and `partialIndexWorker` individually ?